guess_the_number.py

```diff
@@ -17,7 +17,11 @@ def generate_secret(low, high):
 
 def get_guess():
     '''get user's guess'''
-    return int(input('Guess the secret number? '))
+    try:
```

Comment:
Validation appears to be appropriate for the issue.

Reply:
Thank you for your feedback